### ISSUE TITLE
Cannot edit multiple objects

### ISSUE THREAD

**Issue by Duion** [GarageGames/Torque3D#1995]
I remember being able to select and edit the values of multiple objects before in Torque, but it seems to be broken now.
To reproduce: Place multiple objects in the default template, like some rocks, select them all and edit some values like scale and you will notice it will only update the item you selected last.

You really should be able to select and edit multiple objects at once, imaging having placed hundreds of objects and in the end you realize you have to switch collision type to visible mesh or something like that on all of them and it would be just insane to select each one separately and change the value and then repeat it a hundred times, when you just could select all and change it and done.

**Comment by Duion** [comment]
Hm I found that for some things it seems to work, for example if you select multiple audio emitters and change the volume, it will change the volume on all at the same time.

**Comment by dottools** [comment]
Makes sense you can only do multiple edit for the same object type and possibly only for some fields too.

**Comment by Duion** [comment]
I was never trying to edit different object types, it was always the same like static shapes etc.
The question now is where the reason is, why it only works for certain types, since it should work with any object type that has the same fields like coordinates or the same flags, since the only task that needs to be done is to write the same values in the same field in the mission file.
The engine does account for that already, for example if you edit multiple objects with different values in a certain field, the field will show as empty, signalizing that not all objects have the same value and when you enter some value in there, it will apply for all objects.
